hwdb.d/60-keyboard.hwdb

```diff
@@ -283,11 +283,11 @@ evdev:name:Asus WMI hotkeys:dmi:bvn*:bvr*:bd*:svnASUS*:pn*:*
 evdev:name:Eee PC WMI hotkeys:dmi:bvn*:bvr*:bd*:svnASUS*:pn*:*
 evdev:name:Asus Laptop extra buttons:dmi:bvn*:bvr*:bd*:svnASUS*:pn*:*
  KEYBOARD_KEY_6b=f21                                    # Touchpad Toggle
- KEYBOARD_KEY_7c=f20                                    # Remap micmute to f20
+ KEYBOARD_KEY_7c=micmute
 
 # USB keyboard in Asus FX503VD
 evdev:input:b0003v0B05p1869*
- KEYBOARD_KEY_ff31007c=f20                              # Remap micmute to f20
+ KEYBOARD_KEY_ff31007c=micmute
```

Comment:
This is mapped to MICMUTE at the kernel level already since 2018 and before that there was no mapping for this scancode (disallowing override from hwdb). So this entire section can be dropped instead of changing the mapping.